homeassistant/components/homematic/sensor.py

```diff
@@ -96,6 +104,11 @@ def unit_of_measurement(self):
         """Return the unit of measurement of this entity, if any."""
         return HM_UNIT_HA_CAST.get(self._state, None)
 
+    @property
+    def device_class(self):
+        """Return the device class to use in the frontend, if any."""
+        return HM_DEVICE_CLASS_HA_CAST.get(self._state, None)
```

Comment:
`...get(...)` without None. You can also cleanup the others if you want

Reply:
Done